libs/ml/include/ml/core/node.hpp

```diff
@@ -20,16 +20,6 @@
 #include "logging/logging.hpp"
 #include "ml/ops/ops.hpp"
 #include "ml/ops/weights.hpp"
```

Comment:
i think weights include can be moved to source? I don't see any usage of weights in the header